cache/build.gradle

```diff
@@ -1,36 +1,27 @@
-buildscript {
-    tasks.withType(JavaCompile) {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
-    }
-}
-
 plugins {
-    id 'java'
-    id 'org.jetbrains.kotlin.jvm'
+    id 'org.jetbrains.kotlin.multiplatform'
+    id 'kotlinx-atomicfu'
     id 'org.jetbrains.dokka'
 }
 
-dependencies {
-    compileOnly libraries.jsr305
-    testImplementation libraries.junit
-    testImplementation libraries.truth
-}
-
-apply from: rootProject.file("gradle/jacoco.gradle")
-
-apply plugin: 'com.vanniktech.maven.publish'
+kotlin {
+    iosX64()
+    iosArm64()
+    jvm()
+    //iosSimulatorArm64() sure all ios dependencies support this target
 
-targetCompatibility = 1.8
-sourceCompatibility = 1.8
-
-compileKotlin {
-    kotlinOptions {
-        jvmTarget = "1.8"
-    }
-}
-compileTestKotlin {
-    kotlinOptions {
-        jvmTarget = "1.8"
+    sourceSets {
+        commonMain{}
+        commonTest {
+            dependencies {
+                implementation (libraries.kotlinTestJunit)
+                implementation (libraries.coroutinesTest)
+            }
+        }
+        jvmTest{}
     }
 }
+
+//apply from: rootProject.file("gradle/jacoco.gradle")
```

Comment:
Can we keep Jacoco and kmp or is it not supported?

Reply:
I'll take care of this one after the fact, implementor had trouble with jacoco and version changes. This is not going into main so no concern for now

Reply:
Let's remove jacaoco for now and make a ticket to add back in  please